flytekit/core/type_engine.py

```diff
@@ -1676,7 +1677,8 @@ def get_literal_type(self, t: Type[T]) -> Optional[LiteralType]:
         Only univariate Lists are supported in Flyte
         """
         try:
-            sub_type = TypeEngine.to_literal_type(self.get_sub_type(t))
+            s = self.get_sub_type(t)
+            sub_type = TypeEngine.to_literal_type(s)
```

Comment:
revert this. I'm assuming this is a leftover from some debugging you were doing.